README.md

```diff
@@ -728,6 +728,30 @@ In either case:
       end
     ```
 
+* <a name="unless-with-comparator"></a>Avoid `unless` with comparators if you can use `if` with an opposing comparator.<sup>[[link](#unless-with-comparator)]</sup>
```

Comment:
Do we always need to cite rationales or references? In the past we haven't required this. I don't feel we need to require it every time.